### PR TITLE
Update option styles

### LINKED ISSUE #40: Highlight the EQIP option text when on the EQIP page and gray out the other options that are not allowed

### PR DESCRIPTION
**Pull request by ziyue5**
resolves #40

### CODE REVIEW

File: src/components/ProgramDrawer.tsx [hunk 1]
```diff
@@ -181,19 +191,39 @@ export default function ProgramDrawer({ setEQIPChecked }): JSX.Element {
...
+                ) : (
+                    <Typography>CRP: Conservation Reserve Program</Typography>
+                )}
             </MenuItem>
             <MenuItem style={{ whiteSpace: 'normal' }} sx={{ my: 1 }}>
-                ACEP: Argriculture Conservation Easement Program
+                {location.pathname === '/eqip' ? (
+                    <Typography color="gray">ACEP: Argriculture Conservation Easement Program</Typography>
+                ) : (
+                    <Typography>ACEP: Argriculture Conservation Easement Program</Typography>
```
Comment (navarroc) [inline src/components/ProgramDrawer.tsx:211]: Same here, typo in Agriculture

**Review by navarroc — APPROVED** [review]

File: src/components/ProgramDrawer.tsx [hunk 2]
```diff
@@ -168,11 +170,19 @@ export default function ProgramDrawer({ setEQIPChecked }): JSX.Element {
         >
             <Box sx={{ height: 100 }} />
             <MenuItem style={{ whiteSpace: 'normal' }} sx={{ my: 1 }}>
```
Comment (sandeep-ps) [inline src/components/ProgramDrawer.tsx:172]:
```suggestion
            <MenuItem disabled style={{ whiteSpace: 'normal' }} sx={{ my: 1 }}>
```
could be a better option here as it disables the entire sub-menu.

**Review by sandeep-ps — APPROVED** [review]
I will approve this PR for merging. There is a better option to disable the menu. I have suggested this in a comment. Thanks.

File: src/components/ProgramDrawer.tsx (hunk 2 above)
Comment (ziyue5) [reply src/components/ProgramDrawer.tsx:172]: Thanks! I think the menu items should be functional in near future, so would it be better to change their styles in this way?

**Pull request merged**